package.json

```diff
@@ -89,14 +89,14 @@
     "npm-run-all": "4.1.5",
     "postcss-loader": "8.1.1",
     "prettier": "3.2.5",
-    "sass": "1.74.1",
-    "sass-loader": "14.1.1",
+    "sass-embedded": "1.77.5",
```

Comment:
I've swapped to [`sass-embedded`](https://www.npmjs.com/package/sass-embedded) to use the faster Dart native version